src/EssentialsPE/Events/CreateAPIEvent.php

```diff
@@ -14,10 +14,11 @@ class CreateAPIEvent extends PluginEvent{
     /** @var string */
     private $class;
 
-    /**
-     * @param Loader $plugin
-     * @param BaseAPI::class $api
-     */
+	/**
+	 * @param Loader $plugin
+	 * @param string $api
+	 * @internal param $BaseAPI ::class $api
+	 */
```

Comment:
indentation is messed up here, and the final parameter noted there appears not to exist.

Reply:
Fixed